src/ngraph/runtime/intelgpu/intelgpu_backend.cpp

```diff
@@ -1728,11 +1728,11 @@ shared_ptr<runtime::Executable>
 
             const shared_ptr<op::ConvolutionBackpropData> conv_op =
                 static_pointer_cast<op::ConvolutionBackpropData>(op);
-            const Strides& win_stride = conv_op->get_window_movement_strides_backward();
-            const CoordinateDiff& pad_below = conv_op->get_padding_below_backward();
-            const CoordinateDiff& pad_above = conv_op->get_padding_above_backward();
-            const Strides& win_dilation = conv_op->get_window_dilation_strides_backward();
-            const Strides& data_dilation = conv_op->get_data_dilation_strides_backward();
+            const Strides& win_stride = conv_op->get_data_dilation_strides_forward();
+            CoordinateDiff pad_below = conv_op->compute_backward_delta_out_pad_below();
```

Comment:
I don't understand why you removed "&" (and `const`). Is it necessary to do `memvove` here?

Reply:
If I had to guess I would say it is because the function that uses it below did not mark this input as const and a non-const reference is bad style unless you really want that.

Reply:
No, it's because it's returning a computed value; you have the option of computing that value once and stashing it yourself, but using a version stashed in the op isn't a good thing since in the future the op might not be there when you execute.